scipy/spatial/transform/rotation.py

```diff
@@ -629,3 +629,31 @@ def inv(self):
         if self._single:
             quat = quat[0]
         return self.__class__(quat, normalized=True)
+
+    def __mul__(self, other):
+        """Returns the composition of the rotations.
+
+        If `p` and `q` are two rotations, then the composition of 'q followed
```

Comment:
It should be "p followed by q", right?

Reply:
I checked this, and quaternions compose the same way as dcms, i.e. right to left. Check out slide 18 of [this](http://run.usc.edu/cs520-s12/quaternions/quaternions-cs520.pdf) presentation. It seems to corroborate my view. Of course, we could choose a convention for `p * q` to be `p` followed by `q`, which would amount to `_quat_compose(quat_q, quat_p)`.

Reply:
I'm confused, "p * q", means "p then q", means "p followed by q". I.e. it is q that follows p.

I'm not taking about convention changes, simply about language. Am I wrong in this part (English phrasing)?

Reply:
Well, I think we're both a little confused here.
When writing `p * q`, I'm using the same convention that dcms follow. So `p * q` is the same as "q then p", and "p follows q".

Reply:
I understand you now. You mean vector transformations, whereas I mean physical frame rotations, like first we rotate by p and then by q (to get the final frame).

Let's think about it some more, and then update the description.

Reply:
Ah yes, of course. Both are equally valid ways of looking at it.